dbt_common/clients/system.py

```diff
@@ -608,11 +608,35 @@ def rename(from_path: str, to_path: str, force: bool = False) -> None:
     shutil.move(from_path, to_path)
 
 
+def safe_extract(tarball, path=".", members=None, *, numeric_owner=False):
```

Comment:
Can we add type annotations + remove the attributes that aren't actually used (e.g. members=None, *, numeric_owner=False)